scripts/move_emails.py

```diff
@@ -31,14 +31,14 @@ class MoveEmailsApp(generic_app.App):
     required_config.namespace('elasticsearch')
     required_config.elasticsearch.add_option(
         'elasticsearch_class',
-        default='socorro.external.elasticsearch.connection_context.'
+        default='socorro.external.es.connection_context.'
```

Comment:
Same here, this file was a one-time script, we won't ever need it again, feel free to remove it.